test/unit/int_algolia/scripts/algolia/helper/productFilter.test.js

```diff
@@ -29,36 +29,36 @@ describe('productFilter.isInStock', () => {
     test('Master product with in-stock variant is considered in stock', () => {
         const inStockVariant = new VariantMock({ ats: 5 });
         const outOfStockVariant = new VariantMock({ ats: 0 });
-        
+
         const masterProduct = new MasterProductMock({
             variants: [outOfStockVariant, inStockVariant]
         });
-        
+
         const inStock = productFilter.isInStock(masterProduct, 1);
         expect(inStock).toBe(true);
     });
 
     test('Master product with all out-of-stock variants is considered out of stock', () => {
         const outOfStockVariant1 = new VariantMock({ ats: 0 });
         const outOfStockVariant2 = new VariantMock({ ats: 0 });
-        
+
         const masterProduct = new MasterProductMock({
             variants: [outOfStockVariant1, outOfStockVariant2]
         });
-        
+
         const inStock = productFilter.isInStock(masterProduct, 1);
         expect(inStock).toBe(false);
     });
 
     test('Variation group with in-stock variant is considered in stock', () => {
         const inStockVariant = new VariantMock({ ats: 3 });
-        
-        const variationGroup = new MasterProductMock({
+
+        const master = new MasterProductMock({
```

Comment:
We instantiate it with `variationGroup: true` so it's actually a variation group
```suggestion
        const variationGroup = new MasterProductMock({
```

Reply:
Reverted.